assets/base/i18n/en.json

```diff
@@ -1290,6 +1290,7 @@
   "channel_header.rename": "Rename Channel",
   "channel_header.setHeader": "Edit Channel Header",
   "channel_header.setPurpose": "Edit Channel Purpose",
+  "channel_header.mobile.edit": "Edit Channel",
```

Comment:
this is wrong, it should be `mobile.channel_header.edit` or `mobile.channel_info.edit` but the prefix should always be `mobile.`

Change the code accordingly as I sure missed it

Reply:
What is this convention used for? mobile web? I would like to understand this a little better I see there's a group using the mobile prefix just would like to understand this better.

whats the difference between
`something.mobile.text` and
`mobile.something.text`

Reply:
Right, the translation server will pick everything that has a `mobile.` prefix and add it to the mobile project and the rest is from the webapp project, that is why we need to have the `mobile.` prefix